app/github_public_repo/estimate_csv_creator.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+import pandas as pd
+from flask import current_app as app
+
+from app.database_etl.airtable_records_handler import get_all_records, airtable_get_request
+
+# Load env variables for Airtable request
+AIRTABLE_API_KEY = app.config['AIRTABLE_API_KEY']
+AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL = app.config['AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL']
+
+# Get columns that should be pulled into CSV
+headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+data = airtable_get_request(AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL, headers)
+
+# Get csv columns and create csv
+try:
+    # Extract fields
+    records = data['records']
+    fields = [x['fields']['Field Name'] for x in records]
+    logging.info("Successfully retrieved field names from Airtable")
+
+    # Get estimates from Rapid Review: Estimates table for specified fields
+    csv_records = get_all_records(fields)
+
+    # Convert to df, reorder cols alphabetically and as save as csv
+    csv_records_df = pd.DataFrame.from_dict(csv_records)
```

Comment:
Which rows are included here? All rows, or only primary estimates? If all rows, have we included the identifiers that distinguish between two highly similar estimates so folks can easily identify what the differences are? Have we provided the guidance in the README that indicates this structure, and why there are many estimates within each study?

Reply:
ya, will fix this. we will keep the column ordering in: https://airtable.com/appFfYIupTrVQ0HJx/tbliKC06AgJ1EAKLa/viwoxvB5i6kITkaiF?blocks=hide